sharktank/tests/ops/sharded_test.py

```diff
@@ -1460,6 +1460,27 @@ def testSoftmaxSplit(self):
         ops.equal(expected_result, ops.softmax(sharded_tensor, dim=dim + 1))
 
 
+class SumTest(unittest.TestCase):
```

Comment:
There is no test for the `keepdim=True` case. It is not critical but it would be nice to have one.

Reply:
Added